### DISCUSSION TITLE
Stequometry normalization

### DISCUSSION THREAD

**Discussion by AlejandroEngESC** [firemodels/fds#13705] [Q&A]
Dear team, 

If I'm sumilating a fire with an specific stequometry formula, fixing the HRR and the sub-yield; if I divide the stequometry formula by 10, keeping the HRR and the sub-yield, why is my simulation generating less smoke? Since the power that the fire is developing and the ammount of smoke it expels is the same?

**Comment by drjfloyd** [comment]
How have you determined it is generating less smoke? 

&REAC C=10,H=10,O=10,SOOT_YIELD=0.1/

and

&REAC C=1,H=1,O=1,SOOT_YIELD=0.1/

Are not going to give indentical results. One gas has a molecular weight (MW) of 290 g/mol and one 29 g/mol. One will have 10 times the density at the burner surface. The high MW fuel (unless you define properties on SPEC as discussed in the user's guide) willl have a very different specific heat than the low MW fuel. 

If you are not seeing extinction occuring and your fire is well venitlated and the fuel has time to combust before existing the domain in both case, the smoke production integrated over time should be the same. If this is not case, make sure you are using the current release, double check your inputs, and make sure you are making a quantitative measurement of the smoke production (not just rely on it visually in smokeview). If you still have problems post a simple input file that demonstrates the problem.
